NEO-TAT/tat_flutter
# fix: remove unimplemented notification page

## Description

This PR removes the unimplemented notification page from the project. This includes cleaning up associated localization strings, dependencies, and unused code. The notification page was previously a placeholder and is no longer planned for implementation.

## Implementation

- [x] Removed `NotificationPage` widget and its related file (`lib/ui/pages/notification/notification_page.dart`).
- [x] Updated `MainScreen` to exclude the notification page from the navigation structure.
- [x] Deleted the `titleNotification` localization strings from all relevant localization files (`intl_en.dart`, `intl_zh_TW.dart`, `intl_en.arb`, `intl_zh_TW.arb`).
- [x] Cleaned up references to `titleNotification` in the generated localization classes (`S`).
- [x] Verified the removal of references in the codebase to avoid broken dependencies.

### Breaking Changes

- [x] Removed a navigation tab for the notification page, which may affect users if the page was accessible via routes or other manual integrations.

## Testing Instructions

1. **Code Review**
   - Confirm that all references to `NotificationPage` and `titleNotification` have been successfully removed.
   - Verify that there are no remaining imports or calls to the deleted files/classes.

2. **Manual Testing**
   - Build and run the app.
   - Verify that the notification tab no longer appears in the bottom navigation bar.
   - Ensure all other tabs and functionality remain unaffected.

## Additional Notes

- The notification page may be reintroduced in the future if required. The removed implementation is available in the commit history for reference.
